shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,23 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
+  HIGHCONTRAST high_contrast = {sizeof(HIGHCONTRAST)};
```

Comment:
Since this will be used for an in-out parameter, consider for readability:
```cc
// SystemParametersInfo requires cbSize to be set.
HIGHCONTRAST high_contrast = {
  .cbSize = sizeof(HIGHCONTRAST)
};
```